doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -888,6 +888,14 @@ The specific use case of the Google Fast Pair application is indicated by the ch
 In the official `Fast Pair`_ documentation, the `Fast Pair Device Feature Requirements`_ category defines additional requirements for each supported use case, and specifies a list of mandatory, optional, and unsupported Fast Pair features.
 If your product is targeting one of the listed use cases, you must align your accessory firmware to meet these requirements.
 
+If your Google Fast Pair use case is supported by the |NCS|, you can select it using the :kconfig:option:`CONFIG_BT_FAST_PAIR_USE_CASE` Kconfig choice option.
+The selected Kconfig option configures the Fast Pair feature and extensions to satisfy the `Fast Pair Device Feature Requirements`_ for your target use case.
```

Comment:
Consider explicitly mentioning that some of the requirements need to be implemented at application level.

Reply:
`features and extensions`